src/themes/amsterdam/global_styling/mixins/shadow.ts

```diff
@@ -65,7 +65,12 @@ export const euiShadowMedium = (
   ];
 
   if (property === 'filter') {
-    return `filter: ${array.reduce((v, i) => `${v} drop-shadow(${i})`, '')};`;
+    // To prevent Safari from rendering multiple weird shadows we can only use one drop-shadow
+    return `filter: drop-shadow(0 5.7px 9px ${getShadowColor(
+      color,
+      0.2,
+      colorMode
+    )});`;
   } else {
     return `box-shadow: ${array.map((v) => v)};`;
```

Comment:
Just to follow up on the above, if this is an intentional and permanent change, I think we should reflect that in our code so that only box-shadow has multiple lines, and also remove `const array =` on line 60:
```suggestion
    return `box-shadow:
      0 .9px 4px ${getShadowColor(color, 0.08, colorMode)},
      0 2.6px 8px ${getShadowColor(color, 0.06, colorMode)},
      0 5.7px 12px ${getShadowColor(color, 0.05, colorMode)},
      0 15px 15px ${getShadowColor(color, 0.04, colorMode)};`;
```